### DISCUSSION TITLE
Iterate through a file's commit history.

### DISCUSSION THREAD

**Discussion by pinkisemils** [tpope/vim-fugitive#2193] [Q&A]
Hiya, I was wondering if it's possible to, in a fairly efficient fashion, iterate through a file's commit history - but instead of looking at the changes of any one commit (like opening a commit from the output of `:Git log -- %`, but instead just read the file in it's entirety, possibly with another window showing the list of commits.

**Comment by tpope** [comment]
Try `:0Gclog`.
